packages/synapse-react-client/src/components/ChallengeTeamWizard/ChallengeTeamTable.tsx

```diff
@@ -17,26 +22,24 @@ export type ChallengeTeamTableProps = {
 }
 
 // Custom type for the challenge team table that combines fields from the Team and the ChallengeTeam record
-type ChallengeTeamRow = {
-  id: string
-  name: string
-  created: string
-  description?: string
-}
-
-const getTeamRow = (team: Team): ChallengeTeamRow => {
-  return {
-    id: team.id,
-    name: team.name,
-    created: formatDate(dayjs(team.createdOn), 'MM/DD/YY'),
-    description: team.description,
-  }
+type RowData = Team & {
+  hasInvitation: boolean
 }
```

Comment:
The row data included the fetched teams, so just reuse the `Team` type. Additionally, we will fetch open invitations.